### PR TITLE
Topic recognition

### PR DESCRIPTION
**Pull request by xiaofangC**
3D-Prostate image Segmentation using UNet3D implemented in Tensorflow

### CODE REVIEW

**Review by wooboyeong — CHANGES_REQUESTED** [review]
Please remove commit b1b989d - affects someone else's files

**Comment by wooboyeong** [comment]
Code
* Poor segmentation performance
* dice_coef metric might not be working because you didn't one-hot encode the masks
* Might need to train for longer

README
* Please show the input image for the example output

General Comments
* Commit messages OK mostly but could be more descriptive

**Comment by xiaofangC** [comment]
The commit b1b989d has been removed. 
And just a kind advice that projects with same names but case different cause conflicts in some systems, I wasted hours solving conflicts with git. It might be better to remind students maintaining good naming manner when create projects.

**Comment by shakes76** [comment]
## TF/Torch Usage
Adequate use and implementation
## Good Practice (Design/Commenting)
Good spacing and comments
No Header blocks -1
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage 
Module present 
No Data leakage 
Difficulty: Normal -5
## Commit Log
Meaningful commit messages
Progressive commits used
## Documentation
Good Description and Comments 
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
No feedback required
Request Description could be more informative -1
